sources/osgUtil/Composer.js

```diff
@@ -355,48 +426,172 @@ define( [
             this._dirty = false;
         }
     } );
 
 
+    // filter that switch its render target and its input at each frame
+    // allowing to get input for last frame render.
+    Composer.Filter.PingPong = function ( cameraRtt0, rtt0, cameraRtt1, rtt1, fragmentShader, uniforms ) {
+        Composer.Filter.Custom.apply( this, [ fragmentShader, uniforms ] );
+
+        this._cameraRtt0 = cameraRtt0;
+        this._rtt0 = rtt0;
+
+        this._cameraRtt1 = cameraRtt1;
+        this._rtt1 = rtt1;
+
+        this._fragmentName = 'PingPong';
+    };
+
+    Composer.Filter.PingPong.prototype = MACROUTILS.objectInherit( Composer.Filter.Custom.prototype, {
+
+        // Constraints:
+        // - Next Filter: texture unit 0 === rtt0 && texture unit 1 === rtt1
+        // - Previous Filter: the output of the previous filter if any
+        //      will be binded to texture unit 0 of both camera stateset
+        interConnectFilters: function ( composer, i, array ) {
+
+            var filterStateSet = this.getStateSet();
+
+            var st0 = this._cameraRtt0.getOrCreateStateSet();
+            var st1 = this._cameraRtt1.getOrCreateStateSet();
+
+            // copy filter program and uniforms on the 2 cameras
+            st0.setAttributeAndModes( this._program );
+            st1.setAttributeAndModes( this._program );
+
+            if ( this._uniforms ) {
+                for ( var k = 0; k < this._uniforms.length; k++ ) {
+                    st0.addUniform( this._uniforms[ k ] );
+                    st1.addUniform( this._uniforms[ k ] );
+                }
+            }
+
+            st0.addUniform( Uniform.createInt1( 0, 'Texture0' ) );
+            st0.addUniform( Uniform.createInt1( 1, 'Texture1' ) );
+
+            st1.addUniform( Uniform.createInt1( 0, 'Texture0' ) );
+            st1.addUniform( Uniform.createInt1( 1, 'Texture1' ) );
+
+            // copy input on both camera
+            // Composer::Build set the last render into the current filter stateset texture unit 0
+            // we copy that into each camera as Texture unit 0
+            st0.setTextureAttributeAndModes( 0, filterStateSet.getTextureAttribute( 0, 'Texture' ) );
+            st1.setTextureAttributeAndModes( 0, filterStateSet.getTextureAttribute( 0, 'Texture' ) );
+
+            st0.setTextureAttributeAndModes( 1, this._rtt1 );
+            st1.setTextureAttributeAndModes( 1, this._rtt0 );
+
+            // if not the last filter
+            // bind both result to next filter
+            // rtt0 to texture unit 0
+            // rtt0 to texture unit 1
+            if ( i !== array.length - 1 ) {
+
+                // just translate stateset to the next filter
+                var nextSt = array[ i + 1 ].filter.getStateSet();
+
+                nextSt.setTextureAttributeAndModes( 0, this._rtt0 );
+                nextSt.setTextureAttributeAndModes( 1, this._rtt1 );
+
+                nextSt.addUniform( Uniform.createInt1( 0, 'Texture0' ) );
```

Comment:
no need to re create Texture0 Texture1 here too, use the same uniforms created before. Probably we could define globally those uniform in osgjs not sure about the side effect but it would be simpler